assembly/pipeline.py

```diff
@@ -113,6 +113,29 @@ def main():
     #init the output list
     output = []
     
+    #dictionary to store QC PASS/FAIL flags
+    qc_verdicts = {
```

Comment:
Can we use consistent snake_case for dictionary fields?
```
        "multiple_species_contamination":False,
        "same_as_expected_species":False,
        "fastq_contains_plasmids":False,
        "acceptable_coverage":False,
        "acceptable_fastqc_forward":False,
        "acceptable_fastqc_reverse":False,
        "acceptable_quast_assembly_metrics":False,
        "acceptable_busco_assembly_completion": False
```

Reply:
fixed!